lib/charms/mysql/v0/mysql.py

```diff
@@ -224,27 +242,27 @@ def configure_mysql_users(self):
 
         # commands  to create 'root'@'%' user
         create_root_user_commands = (
-            f"CREATE USER 'root'@'%' IDENTIFIED BY '{self.root_password}';",
-            "GRANT ALL ON *.* TO 'root'@'%' WITH GRANT OPTION;",
+            f"CREATE USER 'root'@'%' IDENTIFIED BY '{self.root_password}'",
+            "GRANT ALL ON *.* TO 'root'@'%' WITH GRANT OPTION",
         )
 
         # commands to be run from mysql client with root user and password set above
         configure_users_commands = (
-            f"CREATE USER '{self.server_config_user}'@'%' IDENTIFIED BY '{self.server_config_password}';",
-            f"GRANT ALL ON *.* TO '{self.server_config_user}'@'%' WITH GRANT OPTION;",
-            "UPDATE mysql.user SET authentication_string=null WHERE User='root' and Host='localhost';",
-            f"ALTER USER 'root'@'localhost' IDENTIFIED BY '{self.root_password}';",
-            f"REVOKE {', '.join(privileges_to_revoke)} ON *.* FROM root@'%';",
-            f"REVOKE {', '.join(privileges_to_revoke)} ON *.* FROM root@localhost;",
-            "FLUSH PRIVILEGES;",
+            f"CREATE USER '{self.server_config_user}'@'%' IDENTIFIED BY '{self.server_config_password}'",
+            f"GRANT ALL ON *.* TO '{self.server_config_user}'@'%' WITH GRANT OPTION",
+            "UPDATE mysql.user SET authentication_string=null WHERE User='root' and Host='localhost'",
+            f"ALTER USER 'root'@'localhost' IDENTIFIED BY '{self.root_password}'",
+            f"REVOKE {', '.join(privileges_to_revoke)} ON *.* FROM root@'%'",
+            f"REVOKE {', '.join(privileges_to_revoke)} ON *.* FROM root@localhost",
+            "FLUSH PRIVILEGES",
```

Comment:
I've spend way too much time trying to understand what changed here...

Reply:
removed the `;` from the commands, and instead use `'; '.join(commands)`

Reply:
Oh yeah, forgot to add that I did eventually ;)